lib/open3.rb

```diff
@@ -1068,9 +1095,9 @@ def pipeline_r(*cmds, &block)
   #   Open3.pipeline_r(
   #     ['ruby', '-e', 'print "Foo"'],
   #     ['ruby', '-e', 'print STDIN.read + "Bar"']
-  #   ) do |x, ts|
-  #     puts x.read
-  #     p ts
+  #   ) do |first_stdin, wait_threads|
+  #     puts first_stdin.read
+  #     p wait_threads
   #   end
```

Comment:
I just noticed that both of the examples for `Open3.pipeline_w` is wrong because they show `Open3.pipeline_r`.

Reply:
Fixed,